### PR TITLE
build: remove unused code

### PR DESCRIPTION
**Pull request by Balcan**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://dhis2.atlassian.net/browse/ANDROAPP-

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] 11.X - 13.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/mappers/GraphToTable.kt [hunk 1]
```diff
@@ -169,7 +175,15 @@ class GraphToTable {
                                     R.string.line_listing_max_results,
                                     LINE_LISTING_MAX_ROWS,
                                 ),
-                                style = descriptionTextStyle,
+                                style = TextStyle(
+                                    color = Color(0xFF667685),
```
Comment (andresmr) [inline dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/mappers/GraphToTable.kt:179]: don't we have an explicit color for this in the library?. should we create it?

Comment (Balcan) [reply dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/mappers/GraphToTable.kt:179]: It was like that before but it totally makes sense to change it to a color in the library. I'll check if it already exist

**Review by andresmr — APPROVED** [review]
Amazing!

**Review by ferdyrod — APPROVED** [review]
Excellent!!!

😮 👏🏽 👍🏽

**Pull request merged**